docs/app/pages/components/components-sections/tabs/tabs/tabs.component.html

```diff
@@ -86,11 +86,27 @@ <h4>{{ tab.title }}</h4>
 </uxd-api-properties>
 
 <uxd-api-properties tableTitle="Outputs">
+    <tr uxd-api-property name="activated" type="void">
+        This event is triggered when the tab is activated.
+    </tr>
     <tr uxd-api-property name="select" type="void">
         This event is triggered when the tab is selected.
+        <blockquote>
+            <p><b>Note:</b> This output has been deprecated. Please use <code>activated</code>
```

Comment:
The tabs test page is still using the old output. We should probably update this:

![image](https://user-images.githubusercontent.com/20795331/63359847-5794e600-c365-11e9-89ea-b050b8328550.png)

I also think it would be beneficial to add a karma test that tests both of these outputs to ensure we don't break either of them in the future.